src/dotnet/Fable.Client.Browser/Fable.Client.Browser.fsproj

```diff
@@ -30,8 +28,7 @@
     <Compile Include="../Fable.Compiler/FSharp2Fable.Util.fs"/>
     <Compile Include="../Fable.Compiler/FSharp2Fable.fs"/>
     <Compile Include="../Fable.Compiler/Fable2Babel.fs"/>
-
-    <Compile Include="../Fable.Tools/State.fs"/>
+    <Compile Include="../Fable.Compiler/State.fs"/>
```

Comment:
Fable merges all the files of current and referenced projects into a single project and then [adds the basic compiler args](https://github.com/fable-compiler/Fable/blob/61d44e078e6f6068365ffde11bb460364509441b/src/dotnet/Fable.Tools/ProjectCracker.fs#L403) which always contains the `FABLE_COMPILER` flag so this shouldn't be a problem.